pandas-stubs/core/indexes/datetimes.pyi

```diff
@@ -76,25 +83,26 @@ class DatetimeIndex(DatetimeTimedeltaMixin, DatetimeIndexProperties):
     def dtype(self) -> np.dtype | DatetimeTZDtype: ...
 
 def date_range(
-    start=...,
-    end=...,
-    periods=...,
-    freq=...,
-    tz=...,
-    normalize=...,
-    name=...,
-    closed=...,
+    start: str | DatetimeLike | None = ...,
+    end: str | DatetimeLike | None = ...,
+    periods: int | None = ...,
+    freq: str | BaseOffset | Tick = ...,
```

Comment:
I think Tick is a sub-class of BaseOffset?

Reply:
Yes, good catch.  Fixed in next commit